packages/directives/__tests__/click-outside.test.tsx

```diff
@@ -112,33 +87,29 @@ describe('Directives.vue', () => {
 
 describe('Multiple click-outside directives', () => {
   const firstHandler = vi.fn()
-  const InnerComponent = {
-    template: `<div class="${TRIGGER}" v-click-outside="handler">${AXIOM}</div>`,
-    setup() {
-      return { handler: firstHandler }
-    },
-  }
-
   const secondHandler = vi.fn()
-  const OuterComponent = {
-    setup() {
-      return () => {
-        const triggerNode = withDirectives(h(InnerComponent), [
-          [ClickOutside, secondHandler],
-        ])
-        return h('div', { class: OTHER_CLASS }, triggerNode)
-      }
-    },
-  }
 
   it('should support for multiple directives', async () => {
-    const wrapper = mount(OuterComponent, {
-      global: {
-        directives: {
-          ClickOutside,
+    const wrapper = mount(
+      {
+        setup() {
+          return () => (
+            <div class="other-class" v-click-outside={secondHandler}>
+              <div class="trigger" v-click-outside={firstHandler}>
```

Comment:
## Suggestion

I think it's better to use a **constant** that has already been defined. `OTHER_CLASS`, `TRIGGER`

Reply:
Good catch!